### DISCUSSION TITLE
Notifications insde tmux don't work.

### DISCUSSION THREAD

**Discussion by gachikuku** [kovidgoyal/kitty#8144] [Q&A] [answered]
I'm on macos and when I do `kitten notify "meow"` inside `tmux`, I do not get any notification.

**Comment by kovidgoyal** [comment, marked as answer]
You need to open an issue with tmux requesting they support the
notification protocol.

On Sat, Dec 21, 2024 at 06:54:18AM -0800, gachikuku wrote:
> I'm on macos and when I do `kitten notify "meow"` inside `tmux`, I do not get any notification.
>
> --
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/8144
> You are receiving this because you are subscribed to this thread.
>
> Message ID: ***@***.***>

--
_____________________________________

Dr. Kovid Goyal
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________
